invoke-ai/InvokeAI
# tidy(ui): cleanup

## What type of PR is this? (check all applicable)


## Description

- Fix all typescript issues in frontend - **future PRs much pass checks!**
- Add and use [knip](https://knip.dev/) to remove OODLES of unused code
- Add knip to frontend check workflow, it doesn't fail any but it does run the check
- Move from madge to dpdm for circular dependencies, update workflow
- Remove extraneous model install route - only the heuristic import is used now
- Fix a number of issues in the new model manager UI - mostly related to types

## QA Instructions, Screenshots, Recordings

App should still run. Thanks to all the static analysis tools we use, we can be confident nothing has broken



## Merge Plan

This PR can be merged when approved
